buildSrc/src/main/java/org/elasticsearch/gradle/test/BatsTestTask.java

```diff
@@ -85,6 +108,12 @@ public void runBats() {
             spec.environment(System.getenv());
             spec.environment("BATS_TESTS", testsDir.getAsFile().toString());
             spec.environment("BATS_UTILS", utilsDir.getAsFile().toString());
+            if (pluginsDir != null) {
```

Comment:
Is there any harm in _always_ passing these directories and environment variables? Wouldn't the tests for which these are non-applicable just ignore them?

Reply:
I tried this, but Gradle complained that the directory did not exist, since in several cases the task that sets it up won't have been run.